 docker_image/README.md

```diff
@@ -0,0 +1,18 @@
+
+# Docker SnowShu
+
+A simple image for running SnowShu in a container. 
+
+*Note:* SnowShu uses docker, so when you run the container make sure to add it to the snowshu network with 
```

Comment:
Since snowshu manages another container, the description should probably include a note about making sure the docker socket is included as a volume as well (even though it's included in the example run command)